test/integration/waves.cc

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
```

Comment:
```suggestion
`include <string>`
`include <memory>`
```

Reply:
added in [2068ae5](https://github.com/gazebosim/gz-rendering/pull/908/commits/2068ae5ddb1a8ff8d70bad08de69bb454109f1a7)